packages/prettier-config/package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@schul-cloud/prettier-config",
+  "name": "@hpi-schul-cloud/prettier-config",
   "version": "1.0.0",
-  "description": "This package contains shareable Prettier configuration used by the applications created @schul-cloud.",
+  "description": "This package contains shareable Prettier configuration used by the applications created @hpi-schul-cloud.",
   "main": "index.js",
-  "author": "HPI Schul-cloud (https://schul-cloud.org)",
+  "author": "HPI schul-cloud (https://hpi-schul-cloud.de)",
```

Comment:
Isn't the naming `HPI Schul-Cloud` here and in other cases we have no username/url?

Reply:
name changed

Reply:
HPI schul-cloud --> HPI Schul-Cloud